src/main/java/EventTask.java

```diff
@@ -0,0 +1,30 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class EventTask extends Task{
+
+    private LocalDate time;
+    /**
+     * Constructor for event task object
+     * @param name description of the task
+     * @param isCompleted is it completed or not
+     * @param time time for the task
+     */
+    public EventTask(String name, boolean isCompleted, String time) {
+        super(name, isCompleted);
+        this.time = LocalDate.parse(time);
+    }
+
+    public String getType(){
```

Comment:
Sometimes you have "space" before the "{", sometimes you don't. Maybe you could have a fixed standard? I recommend having "space" before "{".